src/openforms/submissions/models/submission.py

```diff
@@ -297,6 +299,20 @@ def save_registration_status(self, status, result):
 
         self.save(update_fields=update_fields)
 
+    @property
+    def total_configuration_wrapper(self):
+        if not self._total_configuration_wrapper:
+            state = self.load_execution_state()
+            form_steps = state.form_steps
+            if len(form_steps) == 0:
+                return
+
+            wrapper = copy.deepcopy(form_steps[0].form_definition.configuration_wrapper)
+            for form_step in form_steps[1:]:
+                wrapper += form_step.form_definition.configuration
```

Comment:
I don't like the magic method use wit _different_ types, so I'd say to either use each step's configuration_wrapper and sum it, or use a custom method `wrapper.add_to_configuration(step_configuration)`.

I also don't see why the `copy.deepcopy` is needed? The `FormConfigurationWrapper` doesn't implement deepcopy afaik so I'm expecting shallow copies anyway.